dandiapi/api/views/info.py

```diff
@@ -55,12 +57,12 @@ def __init__(self, *args, **kwargs):
     method='GET',
 )
 @api_view()
-def info_view(self):
+def info_view(request):
     api_url = f'{settings.DANDI_API_URL}/api'
     serializer = ApiInfoSerializer(
         data={
             'schema_version': settings.DANDI_SCHEMA_VERSION,
-            'schema_url': schema_url,
+            'schema_url': get_schema_url(request),
```

Comment:
note: here we might actually expand on providing access not only to "dataset" schema but to other schemas and context as well!